src/openrct2/peep/Guest.cpp

```diff
@@ -1571,8 +1571,9 @@ bool Guest::DecideAndBuyItem(Ride* ride, ShopItem shopItem, money32 price)
                     // "I'm not paying that much for x"
                     PeepThoughtType thought_type = static_cast<PeepThoughtType>(
                         shopItem >= ShopItem::Photo2
-                            ? PEEP_THOUGHT_TYPE_PHOTO2_MUCH + static_cast<uint8_t>(shopItem - ShopItem::Photo2)
```

Comment:
This was already a mess, but this does not improve things.

I don't think the adding and subtracting should still be necessary, but @duncanspumpkin can probably tell for sure.
But even then, EnumValue() will be ever so slightly cleaner.

Reply:
Its changed in my PR so don't worry about it.

Reply:
Hm, https://github.com/OpenRCT2/OpenRCT2/pull/13513 already makes great changes to this. Probably fine to leave this as-is and make sure it gets tackled there.